schema/crd.go

```diff
@@ -0,0 +1,42 @@
+package schema
```

Comment:
This is just a suggestion / thinking point..

To be future-thinking i.e. for `openfaas.com/v1alpha2` if there are some big changes, should we have a sub-package or sub-type that represents this specific version?

Then perhaps a whole new set of structs for `openfaas.com/v1beta1` and `openfaas.com/v2alpha1` and so on?

Reply:
I agree maybe have

schema/crd

package